server/data/__testutils/testObjects.ts

```diff
@@ -1,11 +1,12 @@
-import type { OffenderSummary } from '../../@types/assessForEarlyReleaseApiClientTypes'
+import type { AssessmentSummary, OffenderSummary } from '../../@types/assessForEarlyReleaseApiClientTypes'
+import AssessmentStatus from '../../enumeration/assessmentStatus'
 
 const createOffenderSummary = ({
   prisonNumber = 'A1234AB',
   bookingId = 54321,
   forename = 'Jim',
   surname = 'Smith',
-  hdced = '2022-08-01',
+  hdced = '01 Aug 2022',
```

Comment:
**question:** isn't the api returning dates in yyyy-mm-dd format?

Reply:
Yes, it is returning in yyyy-mm-dd.
I am using this as response and updating request with the above format in the test files.